ui/app/mainui/activitycenter/views/ActivityNotificationMessage.qml

```diff
@@ -12,7 +12,7 @@ import utils 1.0
 ActivityNotificationBase {
     id: root
 
-    readonly property bool isOutgoingMessage: notification && notification.message && notification.message.amISender
+    readonly property bool isOutgoingMessage: notification && notification.message && notification.message.amISender || false
```

Comment:
This is pretty much the same as before, right? :)

Reply:
For some notifications, `amISender` is undefined and it raises a qml warning, `|| false` casts it to boolean